cli/cdsctl/tools.go

```diff
@@ -107,7 +107,7 @@ func toolsYamlSchemaRun(v cli.Values) error {
 	results := make([]string, len(types))
 	for i := range types {
 		r := jsonschema.Reflector{
-			AllowAdditionalProperties:  true,
+			AllowAdditionalProperties:  false,
```

Comment:
This was useful to ignore error in yaml when a unknown entry is set. This was set to true to ignore error for custom action where custom names are unknown keys in yaml.

Reply:
Why do we need to change it ?

Reply:
to be able to validate a yaml. To avoid that people enter a key that doesn ’t match the schema.
Today I can type foo: bar in a workflow yaml file, and it’s ok